apache2/waf_logging/waf_log_util.cc

```diff
@@ -5,52 +5,13 @@ using namespace std;
 unordered_map<int, bool> appgw_ruleid_hash;
 
 // This function fills in a waf format message based on modsec input.
-void set_waf_format(waf_format::Waf_Format* waf_format, char* resourceId, char* operationName, char* category, char* instanceId, char* clientIP, char* clientPort, const char* requestUri, char* ruleSetType, char* ruleSetVersion, char* ruleId, char* messages, int action, int site, char* details_messages, char* details_data, char* details_file, char* details_line, const char* hostname, char* waf_unique_id) {
-    waf_format::Properties *properties;
-    waf_format::Details *details;
+waf_logging::waf_log set_waf_format(char* resourceId, char* operationName, char* category, char* instanceId, char* clientIP, char* clientPort, const char* requestUri, char* ruleSetType, char* ruleSetVersion, char* ruleId, char* messages, int action, int site, char* details_messages, char* details_data, char* details_file, char* details_line, const char* hostname, char* waf_unique_id) {
```

Comment:
I think we may even return `std::string` right from this function.
We can fill `waf_logging::waf_log` object just like we do now and return `to_json_string(waf_log);`.

In this case, it also makes sense to rename it to smth like `get_json_log_message()`.

Reply:
will change